src/components/pages/LockScreen.tsx

```diff
@@ -47,11 +54,24 @@ export const LockScreen = (): JSX.Element => {
   const onBiometryLock = (): void =>
     void handleChange(setBiometry, 'biometryLock', webviewIntent)
 
+  const onPinCodeLock = async (pinCode?: string): Promise<void> => {
+    if (!pinCodeEnabled && !pinCode) return setPinModalVisible(true)
```

Comment:
I don't understand how we can reach this code if `pinCodeEnabled` is `false`.

Reply:
Well not sure to understand. If it's false it will be if (true && !pinCode) so we just want to be sure there is no pinCode argument so we can activate the pinCode dialog

Reply:
My point was that if `pinCodeEnabled=false` I don't get how `onPinCodeLock` can be called.

Reply:
sorry not sure to follow here, onPinCodeLock can be called with false or true for pinCodeEnabled